pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "pdp-util"
+version = "2.1.0"
+description = "A package supplying numerous apps for running PCIC's data server"
+authors = ["James Hiebert <hiebert@uvic.ca>"]
+readme = "README.rst"
+homepage = "http://www.pacificclimate.org/"
+repository = "https://github.com/pacificclimate/pdp_util"
+keywords = ["sql", "database", "opendap", "dods", "dap", "data", "science", "climate", "oceanography", "meteorology"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Internet",
+    "Topic :: Internet :: WWW/HTTP :: WSGI",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+packages = [
+    {include = "pdp_util"},
+]
+include = [
+    "pdp_util/data/alpha.png",
+    "pdp_util/data/*.css",
```

Comment:
See [this comment](https://github.com/pacificclimate/pdp_util/issues/47#issuecomment-1834385153), but this can be deferred to another PR.